packages/react-examples/src/react-image/Image/Image.stories.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 import { Image } from '@fluentui/react-image';
+import { teamsLightTheme } from '@fluentui/react-theme';
+import { FluentProvider } from '@fluentui/react-provider';
```

Comment:
```suggestion
```

You can use a decorator instead, it's available now 🐱 

https://github.com/microsoft/fluentui/pull/16868/files#diff-e11bfa97318c042abbf9cd46418de9afda2274a5e8d873e200ee2a5e8bec9832R33-R36